concepticondata/conceptlists/Dawson-1887-731.tsv

```diff
@@ -0,0 +1,732 @@
+Dawson-1887-731-130	130	Ear-rings	770	EARRING	Dress and Ornaments	3	29
+Dawson-1887-731-131	131	Nose-rings			Dress and Ornaments	3	29
+Dawson-1887-731-132	132	Paint (black)			Dress and Ornaments	3	29
+Dawson-1887-731-133	133	Paint (red)			Dress and Ornaments	3	29
+Dawson-1887-731-134	134	Barehead			Dress and Ornaments	3	29
+Dawson-1887-731-135	135	Barefoot	3737	BAREFOOT	Dress and Ornaments	3	29
+Dawson-1887-731-136	136	Naked	590	NAKED	Dress and Ornaments	3	29
+Dawson-1887-731-137	137	Village	930	VILLAGE	Dwellings	4	29
+Dawson-1887-731-138	138	House	1252	HOUSE	Dwellings	4	29
+Dawson-1887-731-139	139	Doorway			Dwellings	4	29
+Dawson-1887-731-140	140	Smoke-hole			Dwellings	4	29
+Dawson-1887-731-141	141	Fire-place	185	FIREPLACE	Dwellings	4	29
+Dawson-1887-731-142	142	Fire	221	FIRE	Dwellings	4	29
+Dawson-1887-731-143	143	Fire-wood	10	FIREWOOD	Dwellings	4	29
+Dawson-1887-731-144	144	Blaze			Dwellings	4	29
+Dawson-1887-731-145	145	A light	630	LIGHT (RADIATION)	Dwellings	4	29
+Dawson-1887-731-146	146	Dead coals			Dwellings	4	29
+Dawson-1887-731-147	147	Ashes	646	ASH	Dwellings	4	29
+Dawson-1887-731-148	148	Smoke	778	SMOKE (EXHAUST)	Dwellings	4	29
+Dawson-1887-731-149	149	Soot	830	SOOT	Dwellings	4	29
+Dawson-1887-731-150	150	Poker			Dwellings	4	29
+Dawson-1887-731-151	151	A seat			Dwellings	4	29
+Dawson-1887-731-152	152	The place where seats are			Dwellings	4	29
+Dawson-1887-731-153	153	Upright post of house			Dwellings	4	29
+Dawson-1887-731-154	154	Main rafter of house			Dwellings	4	29
+Dawson-1887-731-155	155	Mat	195	MAT	Dwellings	4	29
+Dawson-1887-731-156	156	Bed	1663	BED	Dwellings	4	29
+Dawson-1887-731-157	157	Floor	1632	FLOOR	Dwellings	4	29
+Dawson-1887-731-158	158	Ceiling			Dwellings	4	29
+Dawson-1887-731-159	159	Wall	933	WALL (OF HOUSE)	Dwellings	4	29
+Dawson-1887-731-160	160	Lintel			Dwellings	4	29
+Dawson-1887-731-161	161	Opening for window			Dwellings	4	29
+Dawson-1887-731-162	162	Carved post (outside house)			Dwellings	4	29
+Dawson-1887-731-163	163	Carved post (inside house)			Dwellings	4	29
+Dawson-1887-731-164	164	Stairway			Dwellings	4	29
+Dawson-1887-731-165	165	A stone	857	STONE	Dwellings	4	29
+Dawson-1887-731-166	166	Paint mortar	1731	MORTAR BINDER	Dwellings	4	29
+Dawson-1887-731-167	167	Spring	849	SPRING (OF WATER)	Dwellings	4	29
+Dawson-1887-731-168	168	Water	948	WATER	Dwellings	4	29
+Dawson-1887-731-169	169	Bow, of wood	994	BOW	Implements and Utensils	5	30
+Dawson-1887-731-170	170	Bow string	2119	BOW STRING	Implements and Utensils	5	30
+Dawson-1887-731-171	171	Arrow	977	ARROW	Implements and Utensils	5	30
+Dawson-1887-731-172	172	Notch in arrow for string			Implements and Utensils	5	30
+Dawson-1887-731-173	173	Arrow-head of stone			Implements and Utensils	5	30
+Dawson-1887-731-174	174	Arrow feathers			Implements and Utensils	5	30
+Dawson-1887-731-175	175	Quiver			Implements and Utensils	5	30
+Dawson-1887-731-176	176	War-club (stone)			Implements and Utensils	5	30
```

Comment:
Potentially 1763 but I'd have to look at the concrete lexemes again.

Reply:
As we discussed above, 'war-club' is too specific to be mapped CLUB (analogous to the potential mapping of 'war-spear' to SPEAR)

Reply:
Gotcha, sorry, missed that.